projects/storefrontstyles/scss/components/content/tab-paragraph-container/_tab-paragraph-container.scss

```diff
@@ -1,4 +1,51 @@
 $useTabsOnPDP: false !default;
+$useAccordionOnly: false !default;
+
+@mixin accordion {
+  > button {
+    border-top: 1px var(--cx-color-light) solid;
+    font-size: 18px;
+    padding-top: 15px;
+    padding-inline-end: 0;
+    padding-bottom: 0;
+    padding-inline-start: 4%;
+    width: 100%;
+    text-align: start;
+    height: 63px;
+    > span.accordion-icon {
+      &:before {
+        content: '+';
+        margin: 0px 15px;
+        float: right;
+        font-size: 33px;
```

Comment:
font-size: 33px should be change to a variable. `font-size: var(--cx-font-size, 2.063rem);`